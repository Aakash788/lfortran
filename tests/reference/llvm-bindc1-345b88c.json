{
    "basename": "llvm-bindc1-345b88c",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/bindc1.f90",
    "infile_hash": "b894bae3cff84c3540cc34d38c5e3e6c46197db6d93d5f0b48013a06",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-bindc1-345b88c.stdout",
    "stdout_hash": "2b074fd1b82765e759a35a7d0aaa3c912d735a227c79d37e587067c2",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}