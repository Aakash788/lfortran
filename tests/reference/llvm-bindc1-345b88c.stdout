; ModuleID = 'LFortran'
source_filename = "LFortran"

declare i32 @__module_lfortran_intrinsic_iso_c_binding_c_loc(i32*)

define i32 @main(i32 %0, i8** %1) {
.entry:
  call void @_lpython_set_argv(i32 %0, i8** %1)
  %p = alloca void*, align 8
  %x = alloca i32*, align 8
  store i32* null, i32** %x, align 8
  %2 = load i32*, i32** %x, align 8
  %3 = bitcast i32* %2 to void*
  store void* %3, void** %p, align 8
  ret i32 0
}

declare void @_lpython_set_argv(i32, i8**)
